File: docs-examples/example-http-function-kotlin/src/test/kotlin/example/mock/MockData.java
package example.mock;

import java.util.ArrayList;
import java.util.List;

public class MockData {

    public static final List<String> bucketNames = new ArrayList<>();
    public static final List<String> objectNames = new ArrayList<>();
    public static String namespace = "test-namespace";
    public static String tenancyId = "test-tenancyId";
    public static String bucketLocation = "test-location";

    public static void reset() {
        bucketNames.clear();
        objectNames.clear();
        namespace = "test-namespace";
        tenancyId = "test-tenancyId";
        bucketLocation = "test-location";
    }
}

File: docs-examples/example-http-function-kotlin/src/test/kotlin/example/mock/MockData.kt
package example.mock

object MockData {

    @JvmField
    val bucketNames: MutableList<String?> = ArrayList()

    @JvmField
    val objectNames: MutableList<String?> = ArrayList()

    @JvmField
    var namespace: String = "test-namespace"

    @JvmField
    var tenancyId: String = "test-tenancyId"

    @JvmField
    var bucketLocation: String = "test-location"

    @JvmStatic
    fun reset() {
        bucketNames.clear()
        objectNames.clear()
        namespace = "test-namespace"
        tenancyId = "test-tenancyId"
        bucketLocation = "test-location"
    }
}
